### PR TITLE
Courses prototype

### PR DESCRIPTION
**Pull request by jgagne**
### What this PR does

- Adds courses prototype with header/footer for testing (in situ)

### Content

- https://deploy-preview-647--thegymcms.netlify.app/dev/index/
- https://deploy-preview-647--thegymcms.netlify.app/dev/index-auth/

### Preview

gym-prototype-courses-career-skills

### CODE REVIEW

File: dev/prototypes/courses/comp-1.html [hunk 1]
```diff
@@ -0,0 +1,2379 @@
+<link rel="preload" href="{{ site.url }}/dev/prototypes/courses/css/main.css" as="style">
```
Comment (rediris) [inline dev/prototypes/courses/comp-1.html]: Suggest an HTML beautification for this file.

**Comment by rediris** [comment]
Just so I'm clear, we need to be logged in to view the prototype?

File: dev/index.html [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
-</p>
+<link rel="stylesheet" href="{{ site.url }}/css/static-main-content.css">
```
Comment (rediris) [inline dev/index.html]: Suggest adding a cachebuster to all CSS & JS, something like this:

`<link rel="stylesheet" href="{{ site.url }}/css/static-main-content.css?{{ site.time | date:'%s' }}">`

**Comment by jgagne** [comment]
> Just so I'm clear, we need to be logged in to view the prototype?

Yes.

**Review by rediris — APPROVED** [review]

**Pull request merged**